javascripts/src/lib/format.ts

```diff
@@ -15,7 +15,13 @@ export function centsToDollars(cents:string|number|undefined, options:{noCents?:
   return numberWithCommas((centsAsNumber / 100.0).toFixed(options.noCents ? 0 : 2).toString()).replace(/\.00$/,'')
 }
 
-export function dollarsToCents(dollars:string) {
+/**
+ * Convert a string describing a dollar into a number representing the cents. 
+ * @param dollars a string representing an amount in dollars
```

Comment:
Syntax question:

When you add an '@' symbol in front of these words, does that let the library know that: 
a) these lines of code serve like comment lines? 
b) these lines of code should not be run

Reply:
Everything from lines 18-23 is a multi-line comment. The tags like `@param` , `@returns`, etc. are [JSDoc tags](https://jsdoc.app/). They're a standardized way to document functions (and classes) in a Javascript, or in this case Typescript, file. They're machine readable and a lot of tools use them to add richer info to documentation. For example, VSCode would provide additional documentation if you're using the function in another file.

Reply:
😮  Nice! Thank you!